packages/core/src/wallet.ts

```diff
@@ -118,6 +118,62 @@ export enum WalletStatus {
   INACTIVE = "INACTIVE",
 }
 
+export class InactiveMasterWallet {
+  id: string;
+  name: string;
+  blockchain: BlockchainType;
+  henesisKey: Key;
+  status: WalletStatus;
+  createdAt: string;
+  updatedAt: string;
+
+  constructor(
+    id: string,
+    name: string,
+    blockchain: BlockchainType,
+    henesisKey: Key,
+    status: WalletStatus,
+    createdAt: string,
+    updatedAt: string
+  ) {
+    this.id = id;
+    this.name = name;
+    this.blockchain = blockchain;
+    this.henesisKey = henesisKey;
+    this.status = status;
+    this.createdAt = createdAt;
+    this.updatedAt = updatedAt;
+  }
+}
+
+export class ActivatingMasterWallet {
```

Comment:
how about making it unity with `InactiveMasterWallet`? i sugget ActiveMasterWallet

Reply:
When activating master wallet, its status is changed to `CREATING`, not `ACTIVE`. I've added some comments above this.